### PR TITLE
Updated Inelastic Data Analysis Documentation

### LINKED ISSUE #32364: Improve Elwin documentation
As a result of the new workspace load functionality in Elwin the documentation at https://docs.mantidproject.org/nightly/interfaces/indirect/Indirect Elwin & Iqt.html is out of date. This needs to be updated.

**Additional context**
Work resulting from #32044

### PR DESCRIPTION
**Pull request by sf1919**
**Description of work.**
Added an updated screenshot of the Ineleastic Data Analysis GUI to the Inelastic Data Analysis Documentation and the Elwin and I(Q,t) documentation. Also updated the Elwin documentation with details of the new loading facility including a gif demonstrating how to use.

**To test:**
- remove build/docs/doctrees
- build the documents again and check there are no warnings
- Check the information is still correct
- Are the images visible and clearly illustrate the new feature/improvement?
- Grammar/Spelling check

Fixes #32364

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst [hunk 1]
```diff
@@ -18,13 +18,19 @@ Mantid data search paths that contains the sample environment information the
...
-.. interface:: Data Analysis
+.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
   :width: 450
-  :widget: tabElwin
 
 Elwin Options
 ~~~~~~~~~~~~~
 
+File or Workspace
+  Choose to download data from a file or a workspace by using this dropdown menu. See image below for demonstration of how to load files using either option.
```
Comment (DanielMurphy22) [inline docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst:28]: Is `download` the correct word here? I would have thought just 'load' was better but I'm just checking!

File: docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst [hunk 2]
```diff
@@ -18,13 +18,19 @@ Mantid data search paths that contains the sample environment information the
 ELF workspace can also be normalised to the lowest temperature run in the range
 of input files.
 
-.. interface:: Data Analysis
+.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
   :width: 450
```
Comment (DanielMurphy22) [inline docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst:22]:
```suggestion
.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
  :width: 750
  :align: center
```
Can we make this image larger and maybe centred?

File: docs/source/interfaces/indirect/Inelastic Data Analysis.rst [hunk 3]
```diff
@@ -23,7 +23,7 @@ The other two tabs in the interface perform transformations on data and are docu
 
 These interfaces do not support GroupWorkspaces as input.
 
-.. interface:: Data Analysis
+.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
   :width: 650
```
Comment (DanielMurphy22) [inline docs/source/interfaces/indirect/Inelastic Data Analysis.rst:28]: Maybe you want the width and align parameters to be the same on both pages. 650 width I'd say is good as a minimum for this image, I like them large!

**Review by DanielMurphy22 — CHANGES_REQUESTED** [review]
Really good to have these new images and gif! Just a few small things.

File: docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst (hunk 1 above)
Comment (sf1919) [reply docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst:28]: You're right it should be load!

**Review by DanielMurphy22 — APPROVED** [review]
Well done, this looks great! The gif is really impressive!

**Pull request merged**